nutanix/resource_nutanix_ndb_database_scale.go

```diff
@@ -0,0 +1,499 @@
+package nutanix
+
+import (
+	"context"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/terraform-providers/terraform-provider-nutanix/client/era"
+	"github.com/terraform-providers/terraform-provider-nutanix/utils"
+)
+
+func resourceNutanixNDBScaleDatabase() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceNutanixNDBScaleDatabaseCreate,
+		ReadContext:   resourceNutanixNDBScaleDatabaseRead,
+		UpdateContext: resourceNutanixNDBScaleDatabaseUpdate,
+		DeleteContext: resourceNutanixNDBScaleDatabaseDelete,
+		Schema: map[string]*schema.Schema{
+			"database_uuid": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"application_type": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"data_storage_size": {
+				Type:     schema.TypeInt,
+				Required: true,
+			},
+			"pre_script_cmd": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"post_script_cmd": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"scale_count": {
+				Type:     schema.TypeInt,
+				Optional: true,
+			},
+
+			// Computed values
+			"name": {
```

Comment:
The response will be database info right ? So can we import database instance datas source schema here ?